tasks/refresh.rb

```diff
@@ -9,6 +9,7 @@ def output(opts)
     cli_opts = %w[-no-color -json]
     dir = File.expand_path(opts[:dir]) if opts[:dir]
     cli_opts << "-state=#{File.expand_path(opts[:state], dir)}" if opts[:state]
+    cli_opts << "-var-file=#{File.expand_path(opts[:var_file], dir)}" if opts[:var_file]
```

Comment:
If `var_file` supports array of paths we need to handle mapping expansion across those.